lib/triggers.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const { getAPIKey } = require('./userConf');
+const { triggers } = require('../sdk');
+
+async function upsert(triggerName, triggerOptions) {
```

Comment:
nit: this code resides inside `triggers.js` no need to prefix every variable with "trigger"

Reply:
I think it's good in this case, because it's still very short, and it makes it easy to read then copying to conversations

Reply:
also, `name` alone could mean lots of things

`options` could suggest (to me at least) that this options could include things like the `--json` flag, this would not be an issue if you use typescript

Reply:
Good points.